mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -338,6 +337,327 @@ def _load_mesh(self):  # {{{
             refTopDepth, refLayerThickness
         # }}}
 
+    def _compute_moc_climo_analysismember(self):  # {{{
+
+        '''compute mean MOC streamfunction from analysis member'''
+
+        config = self.config
+
+#*********************
+# Needs to be edited from here on
```

Comment:
The following needs to be edited substantially, but we need to keep the option to read from an already computed climatology. I am still figuring out how I do that, hence no much change can be found below :|

Reply:
I can help here if you can describe more specifically what is needed.